src/sentry/silo/patches/silo_aware_transaction_patch.py

```diff
@@ -68,8 +68,14 @@ def determine_using_by_silo_mode(using):
     return using
 
 
+_patched = False
+
+
 def patch_silo_aware_atomic():
-    global _default_on_commit, _default_get_connection, _default_atomic_impl
+    global _default_on_commit, _default_get_connection, _default_atomic_impl, _patched
+
+    assert not _patched
+    _patched = True
```

Comment:
```suggestion
    assert _default_on_commit.__module__ ==  _default_get_connection.__module__ == _default_atomic_impl.__module__ == 'django.db.transaction'
```

can't actually get the whole patch in the editor but this is my point